### DISCUSSION TITLE
How I Can publish two platform(Npm,Github)?

### DISCUSSION THREAD

**Discussion by umutfahriozkan** [community/community#23411] [New to GitHub]
<p>How I can publish two platform?</p>
<p>today.<br>
npm publish<br>
publising to  npm registry</p>
<p>how i can?<br>
npm publish<br>
publising to  github registry<br>
publising to  npm registry</p>
<p>So Is There A Publishing Feature Between Two Platforms?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>There’s a workflow for <a href="https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-to-npm-and-github-packages">publishing to both npm and GitHub</a>.</p>
